standards/technical-approach.md

```diff
@@ -1,12 +1,32 @@
 # Technical Approach Documents
 
-Technical Approach Documents (TADs) serve to help the engineering team document solutions to feature development, share ideas between teams and help highlight potential issues with tasks before they are encountered during the development process.
+Technical Approach Documents (TADs) allow engineers to explore possible solutions to a technical problem before development begins, uncovering potential issues and ultimately landing on a recommended approach through the process.
 
-TADs should strive to describe the whats, hows and whys of the technical problem that is trying to be solved. It should strive to understand the problem at hand in a comprehensive way and explore any challenges so that engineers can be confident in their implementations of solutions. In short writing a TAD should answer any technical questions prior to the start of actual development work, freeing up developers to focus on implementation once the major questions have been resolved.
+## Audience
+
+TADs are ultimately technical documents for a technical audience, not for Product or non-technical stakeholders. They do require approval from the Engineering Leadership Team (ELT), which includes Engineering Managers and the Director of Engineering as [described below](#review-process).
+
+Generally, the sections of the TAD that you'll be writing are for (1) your technical peers, i.e. other engineers who will be reviewing PRs and implementing the feature alonside you, and (2) the Engineering Manager who will be writing the Review Summary section at the end of the review process. The Director of Engineering will be reading the Review Summary section and skimming any other sections of the document that they find helpful.
+
+Since the Engineering Manager might not be intimately familiar with your technical stack or the problem at hand, it is helpful to provide some additional context in the "Problem Definition/Goal" and "Architectural Summary" sections of the template. For example, an iOS engineer might add a brief sentence or two of background information about iOS technologies if the Engineering Manager is not familiar with those technologies.
```

Comment:
@ettore @oliviawongnyc let me know what you think of this in clarifying ELT! basically ELT is (currently) Garvita, Crystal, Mike, and Anthony. Anthony doesn't review these TADs, but Crystal, Mike, and Garvita do.

Reply:
This sounds good to me, so if I understand correctly we should add more context (i.e. assume less knowledge of the stack) in those 2 sections (Problem def/Goal and Architectural Summary) but for the rest our assumption should be the people who routinely review our PRs.  Did I understand well?